docs/install.rst

```diff
@@ -74,6 +74,33 @@ install Kanister with the ``--set controller.updateCRDs=false`` option:
 
 This option lets Helm manage the CRD resources.
 
+Using custom certificates with the Validating Webhook Controller for Blueprints
+==========================================================
+
+By default the Helm chart is configured to automatically generate a
```

Comment:
Let's briefly talk about how installation of kanister also installs a validating webhook for blueprints. And then we can talk about how to provide custom certs to that validating webhook. We should also remove fro blueprints from heading.